File: app/src/test/java/com/aticosoft/appointments/mobile/business/test/common/infrastructure/persistence/TestPersistenceModule.java
package com.aticosoft.appointments.mobile.business.test.common.infrastructure.persistence;

import com.aticosoft.appointments.mobile.business.infrastructure.persistence.PersistenceConfigurer;
import com.aticosoft.appointments.mobile.business.infrastructure.persistence.PersistenceConfigurer.Services;
import com.aticosoft.appointments.mobile.business.infrastructure.persistence.PersistenceModule;

/**
 * Created by rodrigo on 22/08/15.
 */
public class TestPersistenceModule extends PersistenceModule {

    @Override protected PersistenceConfigurer providePersistenceConfigurer(Services services) {
        return new TestPersistenceConfigurer(services);
    }
}

File: app/src/test/java/com/aticosoft/appointments/mobile/business/test/common/infrastructure/persistence/TestPersistenceModule.kt
package com.aticosoft.appointments.mobile.business.test.common.infrastructure.persistence

import com.aticosoft.appointments.mobile.business.infrastructure.persistence.PersistenceConfigurer
import com.aticosoft.appointments.mobile.business.infrastructure.persistence.PersistenceModule

/**
 * Created by rodrigo on 10/09/15.
 */
class TestPersistenceModule : PersistenceModule() {

    override fun providePersistenceConfigurer(services: PersistenceConfigurer.Services) = TestPersistenceConfigurer(services)
}